lib/web_ui/lib/initialization.dart

```diff
@@ -91,7 +91,7 @@ Future<void> webOnlyWarmupEngine({
   } else {
     // Yield control of the bootstrap procedure to the user.
     engine.domWindow.console.debug('Flutter Web Bootstrap: Programmatic.');
-    engine.didCreateEngineInitializer!(bootstrap.prepareEngineInitializer());
+    engine.flutter!.loader!.didCreateEngineInitializer(bootstrap.prepareEngineInitializer());
```

Comment:
Can we refactor this to make the invariants a bit clearer and reduce the number of `!` operators?
```
FlutterLoader? loader = engine.flutter?.loader;
if (loader == null || loader.isAutostart()) {
   ... autostart ...
} else {
  loader.didCreateEngineInitializer(...)
}
```